### ISSUE TITLE
Color Contrast Failure (Dark Mode) in section C.1 Normative references

### ISSUE THREAD

**Issue by clapierre** [mattgarrish/epub-revision#2877]
### Section

https://w3c.github.io/epub-specs/epub34/annotations/#normative-references

### Describe the problem

When you switch the theme from "Light" to "Dark" for the Normative References section.
The color contrast of this section fails WCAG 1.4.3.

### Describe the fix or new feature you propose

I think removing the light blue background like is done in Informal References will resolve this issue.

**Comment by iherman** [comment]
@clapierre you are right but we are not the ones to change that: we are relying on the core W3C style sheets (including for DM).

Copying to @deniak

**Comment by iherman** [comment]
This is not an annotation spec issue, removing the label
